config/webpack.dev.config.js

```diff
@@ -107,7 +107,7 @@ module.exports = Merge.smart(commonConfig, {
       LOGIN_URL: 'http://localhost:18000/login',
       LOGOUT_URL: 'http://localhost:18000/logout',
       CSRF_TOKEN_API_PATH: '/csrf/api/v1/token',
-      REFRESH_ACCESS_TOKEN_ENDPOINT: 'http://localhost:18000/login',
+      REFRESH_ACCESS_TOKEN_ENDPOINT: 'http://localhost:18000/login_refresh',
```

Comment:
Note, this variable already stores the correct route in prod/stage: https://github.com/edx/edx-internal/blob/master/frontends/common/prod_config.yml#L17